src/connection.js

```diff
@@ -130,6 +131,10 @@ class Connection extends EventEmitter {
         this.config.options.enableAnsiNullDefault = config.options.enableAnsiNullDefault;
       }
 
+      if (config.options.enableArithAbort != undefined) {
+        this.config.options.enableArithAbort = config.options.enableArithAbort;
```

Comment:
Please add validation that enableArithAbort is of type Boolean. We don't want someone doing:
config.options.enableArithAbort = 'off' and have that treated as 'on'.
You can follow the pattern for validation for 'config.server' near the top.
Please add a test. You can follow the pattern for the 'badPort' test.